### PR TITLE
feat: add SimpleGatekeeper contract drop Gatekeeper contract

### PR DESCRIPTION
**Pull request by Sokel**
Add SimpleGate - its used in testnet release.

Drop Gatekeeper contract - api changed.

### CODE REVIEW

File: blockchain/source/migrations/4_init_gatekeeper.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    if (network === 'private') {
+        SNM.deployed()
+            .then(function (res) {
+                res.transfer(SimpleGatekeeper.address, 444 * 1e6 * 1e18, { gasPrice: 0 });
+            })
+            .catch(function (err) {
+                console.log(err);
+            });
+    } else if (network === 'rinkeby') {
+    } else {
```
Comment (sshaman1101) [inline blockchain/source/migrations/4_init_gatekeeper.js]: wow, such `else if`, so empty

File: blockchain/source/contracts/SimpleGatekeeper.sol [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    constructor(address _token) public {
+        token = StandardToken(_token);
+        owner = msg.sender;
+    }
+
+    uint256 public transactionAmount = 0;
+    mapping(bytes32 => bool) public paid;
+
+    event PayInTx(address indexed from, uint256 indexed txNumber, uint256 indexed value);
+    event PayoutTx(address indexed from, uint256 indexed txNumber, uint256 indexed value);
```
[the remark this answers is not in the document]
Comment (Sokel) [reply blockchain/source/contracts/SimpleGatekeeper.sol]: I don't think so.

File: blockchain/source/migrations/4_init_gatekeeper.js (hunk 1 above)
Comment (Sokel) [reply blockchain/source/migrations/4_init_gatekeeper.js]: it's ok. Deployer must support this section.

**Comment by Sokel** [comment]
> What is the difference between gatekeeper and simpleGatekeeper?
they have one functional but different implementations and relevant to different releases

**Pull request merged**